osf_tests/test_registration_moderation_notifications.py

```diff
@@ -131,26 +131,8 @@ def test_submit_notifications(self, registration, moderator, admin, contrib, pro
         """
         [REQS-96] "As moderator of branded registry, I receive email notification upon admin author(s) submission approval"
         """
-<<<<<<< HEAD
         with capture_notifications() as notification:
             notify_submit(registration, admin)
-=======
-        # Set up mock_send_mail as a pass-through to the original function.
-        # This lets us assert on the call/args and also implicitly ensures
-        # that the email actually renders as normal in send_mail.
```

Comment:
Same here, conflicts caused by type fix and many occurrences below so I will stop commenting. Great work for looking into these ones.